content/docs/user-guide/testing/lytesttools/example-test-walkthrough.md

```diff
@@ -0,0 +1,16 @@
+---
+description: ' Walk through and explanation of the LyTestTools example test. '
+title: LyTestTools Example Test Walkthrough
+linktitle: LyTestTools Example Test
+---
+
+These are instructions on how to get started with an example test using the LyTestTools (LTT) testing package.
+
+## Prerequisites
+
+* Completed [Getting Started](getting-started.md)
+* A properly configured [Open 3D Engine build](/docs/user-guide/build/)
+* Open the sample test file (`Tools/LyTestTools/tests/integ/sanity_tests.py`)
+* Basic understanding of the [Pytest](https://pytest.org) module
+
+Make sure to complete all of the prerequisite steps above. The provided example test contains in line comments to walk the user through each step. Open the the sample test and follow along for a complete walkthrough.
```

Comment:
? This looks incomplete.

Reply:
The walkthrough has been moved to be in line on the example test itself. This document covers set up and directs the user to  continue the walkthrough in the test.
https://github.com/o3de/o3de/pull/1979/files

Reply:
The page is now a bit curt and awkward to read through. Opening the sample test file should probably not be in 'prerequisites' as it is part of walking through the example. There could also be a brief outline in plain English to suggest which lessons the user should learn from reading the file.

Reply:
Added more description